clients/go/commands/publishMessage_command.go

```diff
@@ -16,121 +16,121 @@
 package commands
 
 import (
-	"context"
-	"fmt"
-	"github.com/zeebe-io/zeebe/clients/go/pb"
-	"github.com/zeebe-io/zeebe/clients/go/utils"
-	"time"
+    "context"
+    "fmt"
+    "github.com/zeebe-io/zeebe/clients/go/pb"
```

Comment:
@menski do we have to somehow specify a rule for formatting like whitespaces instead of tabs  or idk ? :man_shrugging:

Reply:
running `gofmt` generates the expected format